popup.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-plusplus */
+const projects = ['tonic', 'multi-post stories', 'tonic2', 'mlti-poststories2']; /** name copied */
+const workSection = document.getElementById('portfolio');
+const popup = document.getElementById('popup');
+const closePopUp = document.getElementById('closePopUp');
+const openpop = document.getElementsByClassName('openpop');
+const overlay = document.getElementById('overlay');
+/* const myarticles = document.querySelectorAll('.sections'); */
+const viewPortWidth = window.innerWidth;
+const allProjects = [
+  {
+    name: ['tonic', 'multi-post stories', 'tonic2', 'mlti-poststories2'],
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    'featured image': {
+      desktopImages: ['<img class="desktop" src="images/Snapshoot Portfolio - Copy.png" alt=""></img>', '<img class="desktop" src="images/desktop2.opng.png" alt=""></img>', '<img class="desktop" src="images/desktop3.png.png" alt=""></img>', '<img class="desktop" src="images/desktop4.png.png" alt=""></img>'],
+      mobileImages: ['<img class="mobile" src="images/Snapshoot Portfolio (2).png" alt="">', '<img class="mobile" src="images/Snapshoot Portfolio (1).png" alt="">', '<img class="mobile" src="images/Snapshoot Portfolio.png" alt="">', '<img class="mobile" src="images/Snapshoot Portfolio (3).png" alt="">'],
+    },
+    technologies: '',
+    'link to live version': '',
+    'link to source': '',
+  },
+];
```

Comment:
- [ ] Your projects array only has one project object. Kindly add all objects in it so that they can be displayed dynamically. ⛑️